### PR TITLE
Webhook routing

### PR DESCRIPTION
**Pull request by alexandra-plana**
the webhook goes through payload and updates both the feed and and the task status in the db

### CODE REVIEW

File: server/src/middlewares/checkPR.middleware.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { Request, Response, NextFunction } from 'express';
 
 import { Subject } from 'rxjs';
 export const newHook = new Subject();
+export const newLog = new Subject();
```
Comment (thaistcosta) [inline server/src/middlewares/checkPR.middleware.ts:5]: do you mind if I change the name?

**Review by thaistcosta — APPROVED** [review]

**Comment by thaistcosta** [comment]
Niccceee Chica ! Gonna start working on your files ❤️

File: server/src/models/Feed/feed.routes.ts [hunk 2]
```diff
@@ -1,11 +1,11 @@
-import { Router } from 'express';
-import { updateFeed } from './feed.controllers';
-import { cleanData } from './../../middlewares/clean.middleware';
-import { sendEvent } from '../../middlewares/feed.middleware';
+// import { Router } from 'express';
```
[the remark this answers is not in the document]
Comment (alexandra-plana) [reply server/src/models/Feed/feed.routes.ts:1]: i uncommented it on mine sorry

File: server/src/middlewares/checkPR.middleware.ts (hunk 1 above)
Comment (alexandra-plana) [reply server/src/middlewares/checkPR.middleware.ts:5]: feel free S2

**Pull request merged**